.size-limit.json

```diff
@@ -2,14 +2,14 @@
   {
     "name": "zx/core",
     "path": ["build/core.cjs", "build/util.cjs", "build/vendor-core.cjs"],
-    "limit": "76 kB",
+    "limit": "77 kB",
     "brotli": false,
     "gzip": false
   },
   {
     "name": "zx/index",
     "path": "build/*.{js,cjs}",
-    "limit": "804 kB",
+    "limit": "808 kB",
```

Comment:
4Kb? rm `build` dir and measure again.

Reply:
Maybe  esbuild delete the `build` dir before the new build?